apps/services/auth/personal-representative/infra/personal-representative.ts

```diff
@@ -42,10 +42,18 @@ export const serviceSetup =
           prod: 'IS/GOV/5402696029/Skatturinn/ft-v1',
         },
         COMPANY_REGISTRY_REDIS_NODES: REDIS_NODE_CONFIG,
+        SYSLUMENN_HOST: {
+          dev: 'https://api.syslumenn.is/staging',
+          staging: 'https://api.syslumenn.is/staging',
+          prod: 'https://api.syslumenn.is',
+        },
+        SYSLUMENN_TIMEOUT: '10000',
```

Comment:
Same comment regarding staging and timeout

Reply:
Fixed.